VimeoNetworking/Sources/KeychainStore.swift

```diff
@@ -55,20 +55,20 @@ final class KeychainStore
      
      - throws: an error if saving failed
      */
-    func setData(data: NSData, forKey key: String) throws
+    func setData(_ data: NSData, forKey key: String) throws
```

Comment:
I see what you mean. Although I think it seems ambiguous from the call site as 
```Swift
self.keychainStore.set(someData, forKey: someKey)
```
It works really well if you've named your `NSData` variable well, but I could see getting into a weird naming situation. On the other hand…
```Swift
self.keychainStore.set(data: someData, forKey: someKey)
```
… _does_ seem a little redundant. But I kind of like having the clarity a bit more than the brevity.